.gitignore

```
@@ -16,3 +16,4 @@ cli/winresources/rsrc_amd64.syso
 /docs/yaml/gen/
 coverage.txt
 profile.out
+./vndr.log
```

I think either

```suggestion
/vndr.log
```

Or (to match the other ones, although they're not "strictly" correct if the intent was to only ignore at the root

```suggestion
vndr.log
```